test/unit/Cardano/Wallet/Primitive/ModelSpec.hs

```diff
@@ -127,7 +130,8 @@ prop_applyBlockBasic s =
         in
             (ShowFmt utxo === ShowFmt utxo') .&&.
             (availableBalance wallet === balance utxo') .&&.
-            (totalBalance wallet === balance utxo')
+            (totalBalance wallet === balance utxo') .&&.
+            (getTxMetas wallet === mempty)
```

Comment:
Note that the test still fails because of this last assertion, which is actually wrong (hence the failure). Actually, we will collect tx metas as part of the block application (because some of the addresses in the blocks are ours). Not sure if we can (and should) do any assertions in this particular prop on tx metas. This prop is about comparing the UTxO and balance of the basic model with our model (to make sure that our additions don't break the base logic). 

For tx metas, they are probably some other properties we would like to check (like, if the UTxO isn't empty, the tx meta shouldn't be empty too..) or maybe a few more useful one. I haven't gave it many thoughts.